restli-client/src/main/java/com/linkedin/restli/client/Request.java

```diff
@@ -26,6 +26,7 @@
 import com.linkedin.jersey.api.uri.UriTemplate;
 import com.linkedin.restli.common.HttpMethod;
 import com.linkedin.restli.common.ResourceMethod;
+import com.linkedin.restli.common.ResourceMethodIdentifierGenerator;
```

Comment:
Are you adding this class? I don't see it in the existing code or in this PR

Reply:
Yes. Thanks, I had to reclone off my fork when I couldn't commit directly - and I missed adding these new files.